app/src/components/NotFound.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { styles } from '../styles/Home';
+
+class NotFound extends Component {
+    constructor(props) {
+        super(props);
+        this.setState({});
```

Comment:
not needed

Reply:
What would you like to have displayed then for the home page or our 404 ? 
we need to display something and these are the current components for that.

Reply:
or did you mean the empty  state? So you would want this to be a functional component?

Reply:
i meant the empty state